charts/wharf-helm/templates/web.yaml

```
@@ -102,6 +102,23 @@ data:
       },
       "UpdateLatency": 20000,
       "UpdateFrequency": 30000,
+      {{- with .Values.web.oidc }}
+      "oidcConfig": {
+          "authority": "{{ .authority }}",
+          "redirectUrl": "{{ (printf "https://%s" $.Values.global.url) }}",
+          "postLogoutRedirectUri": "{{ (printf "https://%s" $.Values.global.url) }}",
```

Excess parenthases.

Alos, I think it'd be nice to be able to override these fields if the user was so inclined. Such as:

```suggestion
          "redirectUrl": "{{ .redirectUrl | default (printf "https://%s" $.Values.global.url) }}",
          "postLogoutRedirectUri": "{{ .postLogoutRedirectUri | default (printf "https://%s" $.Values.global.url) }}",
```

https://docs.helm.sh/docs/chart_template_guide/function_list/#default

This would also need the fields to be added to `values.yaml`. I think having them default to `null` in `values.yaml` would be a good idea so this usage of `.global.url` is used.

When specifying the fields in `values.yaml` you prefix the comment with `(string)` to override the type, like so:

```yaml
web:
  oidc:
    # -- (string) The redirect URL defined on the Security Token Service.
    redirectUrl:
```

https://github.com/norwoodj/helm-docs#nil-values

---

Whats the usecase for being able to set these values in our case though? I dont think there is one. If it just goes straight to the default url thats gonna be perfect right?

---

Our use case isn't the only use case. In this case its just a few lines more to allow others to have wacky setups if they so want to, while still setting reasonable defaults.